app/src/main/java/com/example/trempel/CategoryRepository.kt

```diff
@@ -0,0 +1,30 @@
+package com.example.trempel
+
+import com.example.trempel.network.CategoryDomainModel
+import com.example.trempel.network.model.CategoryModelItem
+import com.example.trempel.network.service.CategoryService
+import com.example.trempel.network.toCategories
+import io.reactivex.Single
+import io.reactivex.android.schedulers.AndroidSchedulers
+import io.reactivex.schedulers.Schedulers
+import javax.inject.Inject
+
+internal class CategoryRepository @Inject constructor(val retrofitService: CategoryService) {
```

Comment:
can we have `val retrofitService` as private?

Reply:
done